sdk/communication/chat/src/androidTest/java/com/azure/communication/chat/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.azure.communication.chat;
+
```

Comment:
I noticed the other (unit) test will be deleted, will we do that for this one as well?

Reply:
deleted now